### PR TITLE
Adding CI support

### PR DESCRIPTION
**Pull request by Petermarcu**
Moving NuGet.config to the root so restoring and building from the root works on a clean machine.
Adding AppVeyor.yml so my simple CI will run.

### CODE REVIEW

File: Decent.Minecraft.Client/WaterLevel.cs [hunk 1]
```diff
@@ -1,10 +1,12 @@
 ﻿namespace Decent.Minecraft.Client
 {
     /// <summary>
-    /// Species of wood.
+    /// The level of a liquid (ex: Water Level)
     /// </summary>
     public enum Level : byte
     {
+        ///<summary>Source means a block at the highest level setting and originates liquid flow. 
+        /// Only liquid source blocks for can be collected in a bucket. </summary>
```
Comment (bleroy) [inline Decent.Minecraft.Client/WaterLevel.cs]: That "for" seems to be out of place.

**Comment by Petermarcu** [comment]
Fixed the typo and also noticed that I didn't rename the cs file when I changed the Type name to be more generic.

**Comment by bleroy** [comment]
Ah but you also introduced an appveyor file. Can you .gitignore that?

**Comment by Petermarcu** [comment]
That was an intentional add. It's supposed to be checked in. Gitignoring it would make it never be in the repo and its what tells the CI how to run. Any objection to having an AppVeyor CI running? I have it hooked up to run on my fork verifying everything I push there. It already found the NuGet.config problem I also fixed in this PR :)

**Comment by bleroy** [comment]
Mmm. OK, sure, why not. I'd have preferred a separate changeset :D

**Comment by Petermarcu** [comment]
It is a seperate one. Its this one. https://github.com/bleroy/minecraft.client/pull/25/commits/61692ffdb5e280f10bfbc6e1a6b6c6acf73fd08f

**Comment by Petermarcu** [comment]
The PR included 3 different commits.

**Comment by bleroy** [comment]
Apologies then :)

**Comment by Petermarcu** [comment]
Everytime I make progress I push another commit and they pile up until you have time to review and merge :)

**Comment by bleroy** [comment]
Yup, just me not paying attention.

**Comment by Petermarcu** [comment]
No problem, the CI still needs work but it was a scenario I wanted to work through and thought it may add value :)

**Pull request merged**